src/com/palmergames/bukkit/towny/command/PlotCommand.java

```diff
@@ -1786,6 +1770,10 @@ private boolean handlePlotGroupCommand(String[] split, Player player) throws Tow
 						TownyMessaging.sendMsg(player, Translatable.of("msg_set_group_type_to_x", type));
 						
 					})
+						.setCost(new ConfirmationTransaction(cost,
+								resident.getAccount(),
+								String.format("Plot group (" + townBlock.getPlotObjectGroup().getTownBlocks().size() + ") set to %s", type),
+								Translatable.of("msg_err_cannot_afford_plot_set_type_cost", type, TownyEconomyHandler.getFormattedBalance(cost))))

```

Comment:
The cost here will need to be calculated based off of when the confirmation is confirmed, not when it is sent, since that would allow players to pay less if they start adding plots to a group.

Some sort of Supplier<Double> method that allows us to get the cost when it's confirmed might be nice